test/number.js

```diff
@@ -387,5 +387,31 @@ describe('Number', function () {
       expect(Number('0o11')).to.equal(9);
       expect(Number({ toString: function () { return '0o12'; }, valueOf: function () { return '0o13'; } })).to.equal(11);
     });
+
+    it('should produce NaN', function () {
```

Comment:
these tests are failing but it's very hard to figure out which line is failing.

Perhaps `expect(String(Number(…))).to.equal('NaN')` would be better? I don't think we need to worry about testing that `Number()` returns a number :-)